spring-resteasy/pom.xml

```diff
@@ -186,29 +191,132 @@
     </dependencies>
 
```

Comment:
We need also plugin management to be added, please see https://docs.google.com/document/d/1GFnZrnlwRkwC0HfziRws2UUWR99KdaQayA7Ei12SGN4/edit#heading=h.rcftxwtudvl5

Reply:
@emmartins I have added the plugin management section, not sure I add it correctly or not. In addition I found the `batch-proessing`  example doesn't have this section either:

- https://github.com/wildfly/quickstart/blob/main/batch-processing/pom.xml

Shall I add the section their too?

Reply:
Since this is not a MicroProfile quickstart you don't need the wildfly-jar plugin

Reply:
Wrt batch-processing it was fixed yesterday :-)